### PR TITLE
User rating hotel

### PR DESCRIPTION
**Pull request by devtuananh**
https://dev.framgia.com/issues/184357

### CODE REVIEW

**Comment by devtuananh** [comment]
ready

File: app/models/rating.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class Rating < ApplicationRecord
   belongs_to :user
-
+  scope :by_latest, ->{select("ratings.*, users.name").joins("INNER JOIN users ON users.id = ratings.user_id ").order("ratings.created_at desc")}
```
Comment (huongnguyenmta) [inline app/models/rating.rb]: => `select("ratings.*, users.name").joins(:ratings).order created_at: desc`

**Comment by huongnguyenmta** [comment]
commented

File: app/models/rating.rb (hunk 1 above)
Comment (devtuananh) [reply app/models/rating.rb]: C ơi, em muốn ratings join users chị ạ

**Comment by devtuananh** [comment]
ready

**Comment by mdmahmudurrahman** [comment]
commented
This pr can be (merged) 
CC: [To:2076564] Nguyen Huy Hung C (93)
CC: [To:1818111] Nguyen Thi Huong (94)

**Pull request merged**